docs/Recursion/GrayCodeGeneration.md

```diff
@@ -0,0 +1,94 @@
+---
+id: Gray Code Generation
+title:Gray Code Generation Using Recursion 
```

Comment:
replace `title:Gray Code Generation Using Recursion` with `title: Gray Code Generation Using Recursion` because you need space between